### DISCUSSION TITLE
unable to find a way to invoke 'ExperimentalService :: stub :: bulkImportRelationships'  method using Java SDK

### DISCUSSION THREAD

**Discussion by pradeepreddy-anantha** [authzed/spicedb#1998] [Q&A]
Hi,
I am new to Authzed SDK. Exploring bulk Import of relationships. But couldn't find 'bulkImportRelationships' method on ExperimentalServiceGrpc stub. If anyone has code snippet for this feature please share. Thanks.

Ex: ExperimentalServiceBlockingStub stub = ExperimentalServiceGrpc.newBlockingStub(channel).withCallCredentials(bearerToken);
_**stub.bulkImportRelationships(request); --> not found**_

https://buf.build/authzed/api/docs/main:authzed.api.v1#authzed.api.v1.ExperimentalService

**Comment by vroldanbet** [comment]
Hi @pradeepreddy-anantha,

bulk import is not available with the blocking stub. I _believe_ the reason for that is because it is a bidirectional gRPC streaming API.

Here is an example on how to use it: https://github.com/authzed/authzed-java/blob/a8ae90a3f8f008ab6e3ea71299064a7b83938981/src/intTest/java/V1ClientTest.java#L212-L239
